ansible/roles/satellite-installation/tasks/satellite_installation.yml

```diff
@@ -2,9 +2,13 @@
 - name: Colleting host name
   shell: hostname
   register:  hostname_output
```

Comment:
you could use the `ansible_hostname` fact

Reply:
Changed